logprep/ng/processor/list_comparison/processor.py

```diff
@@ -81,14 +81,20 @@ def _list_comparison(self, rule: ListComparisonRule, event: dict) -> tuple[list[
         the result (key) and a list of filenames pertaining to said result (value).
         """
 
+        # get value that should be checked in the lists
```

Comment:
I personally would prefer to not have this kind of comment and instead name the variable `field_value_to_be_checked` (or sth.) instead. I know this is kind of explicit and due to this longer expressions can feel clunky. The advantage however is that typically this kind of documentation ages better and makes the code more readable because you don't have to keep the mapping between variable name and comment in your head